src/datasets/data_files.py

```diff
@@ -73,10 +73,10 @@ class Url(str):
     Split.TRAIN: ["**"],
 }
 
-ALL_SPLIT_PATTERNS = [SPLIT_PATTERN_SHARDED]
+ALL_SPLIT_PATTERNS = [SPLIT_PATTERN_SHARDED]  # TODO: should this be changed?
```

Comment:
I think it's fine this way. This corresponds to the patterns that contain custom split names (other than train/dev/test)